### PR TITLE
setting firewall

### PR DESCRIPTION
**Pull request by schubi2**

### CODE REVIEW

File: spec/lvm.rb [hunk 1]
```diff
@@ -15,6 +15,11 @@
     run_test_script("ntp.sh")
   end
 
+  # bnc #897129
+  it "sets firewall with exisiting network (keep_installed)" do
```
Comment (mchf) [inline spec/lvm.rb]: I'd refer to whole attribute name here (keep_install_network).

**Comment by mchf** [comment]
one NP otherwise LGTM

File: spec/lvm.rb (hunk 1 above)
Comment (schubi2) [inline spec/lvm.rb]: Good point...

**Comment by schubi2** [comment]
Thanks !

**Pull request merged**